authentication/account/service/tenant_whitebox_test.go

```diff
@@ -108,9 +108,14 @@ func (s *TestTenantServiceSuite) TestDelete() {
 	err := s.ts.Delete(ctx, identityID)
 	require.NoError(s.T(), err)
 
-	// Fail if returned anything but No Contented
+	// OK if returned Not Found
```

Comment:
shouldn't you add a test with a non-`204`/non-`404` (eg: `500`) status code to check that an error is returned?

Reply:
Already in place - https://github.com/fabric8-services/fabric8-auth/pull/836/files#diff-b2589ca644bffe667c463b4d566aee6aR116

Reply:
ah ok, thanks! (frankly, subtests make it easier to distinguish the use-cases :P )